recipes/longphase/build.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -xe
+
+# https://bioconda.github.io/troubleshooting.html#zlib-errors
+export CFLAGS="${CFLAGS} -I$PREFIX/include"
```

Comment:
```suggestion
export CPPFLAGS="${CFLAGS} -I$PREFIX/include -L$PREFIX/lib"
```
Removing `zlib` from the `build` deps is caused due to the non-standard handling in the Makefile:
1. https://github.com/twolinin/longphase/blob/v1.7.3/Makefile#L6
   1.1. the `CFLAGS` var is not used at all.
   1.2. The C++ var is named `CPPFLAGS` instead of `CXXFLAGS`. `CPP` is the pre-processor
2. https://github.com/twolinin/longphase/blob/v1.7.3/Makefile#L43 - here the LDFLAGS are not passed at all. That's why I added `-L$PREFIX/lib` to `CPPFLAGS`